src/assignments/mars-rover.rst

```diff
@@ -0,0 +1,137 @@
+Assignment #3: Mars Rover
+=========================
+This task is going to put your unit tests, modules, and exceptions knowledge to use
+by writing tests and classes for the Mars rover named Curiosity.
+
+.. figure:: figures/curiosity-rover-selfie.jpg
+   :alt: Image of Curiosity rover taken by the rover on Mars.
+
+   Selfie of the Curiosity on Mars.
+
+
+Requirements
+------------
+
+#. Fork the `Mars rover starter repl.it <https://repl.it/@launchcode/mars-rover-starter>`_
```

Comment:
The instructional comments on this repl don't make sense.

Reply:
I updated them, lmk if they make more sense.